src/transformers/models/bit/modeling_bit.py

```diff
@@ -620,7 +658,7 @@ def forward(
         )
 
 
-# Copied from transformers.models.resnet.modeling_resnet.ResNetPreTrainedModel with ResNet->Bit,resnet->bit
+# Copied from transformers.models.resnet.modeling_resnet.ResNetPreTrainedModel with ResNet->Bit,resnet->resnetv2
```

Comment:
This won't work when running make fix-copies, as the base model prefix will be set to "resnetv2", it should be "bit"

Reply:
Can you undo this change?

Reply:
fixed in 1a0ed12